Formula/i/ivykis.rb

```diff
@@ -1,8 +1,8 @@
 class Ivykis < Formula
   desc "Async I/O-assisting library"
   homepage "https://sourceforge.net/projects/libivykis/"
-  url "https://github.com/buytenh/ivykis/archive/refs/tags/v0.42.4-trunk.tar.gz"
-  sha256 "b724516d6734f4d5c5f86ad80bde8fc7213c5a70ce2d46b9a2d86e8d150402b5"
+  url "https://github.com/buytenh/ivykis/archive/refs/tags/v0.43-trunk.tar.gz"
+  sha256 "29174b99f45a39bea13f79079d5e17dd58bb800f62e78b44de044715d13d4bdb"
   license "LGPL-2.1"
```

Comment:
https://github.com/buytenh/ivykis/blob/2411ae35af40ccccb77fb9ffc61ee9a9966ad876/src/iv_avl.c#L1-L19

```suggestion
  license "LGPL-2.1-only"
```